lib/Twig/Loader/Filesystem.php

```diff
@@ -74,7 +74,7 @@ public function getNamespaces()
     public function setPaths($paths, $namespace = self::MAIN_NAMESPACE)
     {
         if (!is_array($paths)) {
-            $paths = array($paths);
+            $paths = (array) $paths;
```

Comment:
true, but here it does, `$paths` must a string at this point (as hinted through the PHPDoc), passing something else will cause problems when later on `addPath` is called